ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -349,6 +349,177 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb-subnets-day2
+- as: aws-ipi-byo-subnet-role-isolate-lb-mini-perm-arm-f14
+  cron: 56 16 5,19 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_SUBNETS_COUNT: "1"
+      ASSIGN_ROLES_TO_SUBNETS: "yes"
+      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      SUBNET_ROLES: |-
+        [
+          [
+            {
+              "private": "ClusterNode",
+              "public": "IngressControllerLB BootstrapNode"
+            },
+            {
+              "private": "ControlPlaneInternalLB",
+              "public": "ControlPlaneExternalLB"
+            }
+          ]
+        ]
+      ZONES_COUNT: "1"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-byo-subnets
+- as: aws-ipi-byo-subnet-role-isolate-lb-arm-f28-destructive
+  cron: 1 12 7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_SUBNETS_COUNT: "1"
+      ASSIGN_ROLES_TO_SUBNETS: "yes"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      SUBNET_ROLES: |-
+        [
+          [
+            {
+              "private": "ClusterNode",
+              "public": "IngressControllerLB BootstrapNode"
+            },
+            {
+              "private": "ControlPlaneInternalLB",
+              "public": "ControlPlaneExternalLB"
+            }
+          ]
+        ]
+      ZONES_COUNT: "1"
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-byo-subnets
+- as: aws-ipi-byo-subnet-role-share-lb-mini-perm-arm-f14
```

Comment:
can we merge this job to `aws-ipi-usertags-custom-sg-mini-perm` ?

Reply:
which one?

Reply:
add `role-share-lb` to `aws-ipi-usertags-custom-sg-mini-perm-arm-f7` in openshift-openshift-tests-private-release-4.19__multi-nightly.yaml to save adding the one.

Reply:
@jianlinliu due to the job name length restriction, the name would be `aws-ipi-byo-subnet-role-share-lb-tags-sg-arm-f28-destructive` and `aws-ipi-byo-subnet-role-share-lb-tags-sg-mini-perm-arm-f14`

Reply:
I am okay with that.